src/view/pages/statement/components/SimilarStatementsSuggestion/SimilarStatementsSuggestion.tsx

```diff
@@ -114,36 +115,53 @@ export default function SimilarStatementsSuggestion({
 			) : (
 				<main className='similarities'>
 					<header className='similarities__header'>
-						{currentStep === 2 ? (
-							<div className='similarities__header__top'>
+						{currentStep === 1 ? (
+							<div className='similarities__header__top__stepTwo'>
 								<button
-									className='similarities__header__top__closeButton'
+									className='similarities__header__top__stepTwo__closeButtonStepTwo'
 									onClick={() => setShowModal(false)}
 								>
-									<ArrowLeftIcon />
+									<CloseIcon />
 								</button>
-								<img src={illustration02} alt='Similarities illustration' />
+								<img src={illustration01}
+									alt='Similarities illustration'
+									style={{ width: '8rem' }} />
 							</div>
-						) : (
-							<div className='similarities__header__top'>
+						) : (currentStep === 0 || currentStep === 3? (
+							<div className='similarities__header__top__stepOne'>
 								<img
 									src={illustration}
 									alt='Similarities illustration'
-									style={{ width: '40%' }}
+									style={{ width: '19.5rem' }}
 								/>
 								<button
-									className='similarities__header__top__closeButton'
+									className='similarities__header__top__stepOne__closeButtonStepOne'
 									onClick={() => setShowModal(false)}
 								>
 									<CloseIcon />
 								</button>
 							</div>
-						)}
-						<div className='similarities__header__types'>
-							<div className='type'>
-								<AddQuestionIcon />
-								<h2 className={isQuestion ? 'marked' : 'unmarked'}>Question</h2>
+						) : (currentStep === 2 ? (
+							<div className='similarities__header__top__stepThree'>
+								<img
+									src={illustration02}
+									alt='Another illustration'
+									style={{ width: '19rem' }}
+								/>
+								<button
+									className='similarities__header__top__stepThree__arrowButtonStepThree'
+									onClick={() => setShowModal(false)}
```

Comment:
Clicking the back arrow currently closes the modal, but it should navigate to the previous step instead. Could you adjust the behavior so that it moves back a step rather than closing the modal?